hle-survey/transformation/src/main/scala/org/broadinstitute/monster/dap/HealthTransformations.scala

```diff
@@ -68,8 +67,42 @@ object HealthTransformations {
       None
     }
 
+  def mapEyeDisease(rawRecord: RawRecord): Iterable[HlesHealthCondition] =
+    if (rawRecord.getBoolean("hs_dx_eye_yn")) {
```

Comment:
What about the congenital cases?

Reply:
There are congenital cases?! WHOOPS

Reply:
Oh my goodness I didn't realize there was a hs_cg_eye_disorders_yn